src/content/errors/04014.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "04014"
+slug: "04014"
+description: NG04014 - Invalid configuration of route
+author: Nelson Gutierrez
+authorTwitter: nelsongutidev
+date: Oct 3, 2023
+---
+
+# NG04014
+
+`NG04014` is a runtime error that indicates there is an invalid configuration of the routes. It is aliased as `INVALID_ROUTE_CONFIG` in the `ErrorCode` enum.
+
+### Error Information Examples
+
+```text
+Error NG04014: Invalid configuration of route 'your-route': One of the following must be provided: component, loadComponent, redirectTo, children or loadChildren.
+
+Error NG04014: Invalid configuration of route 'your-route': redirectTo and canActivate cannot be used together. Redirects happen before activation so canActivate will never be executed
+```
+
+### Potential Causes and Solutions
+
+#### 1. Malformed Route Configuration
+
+**Cause**: This error may occur if a route configuration is undefined or contains syntax errors.
+
+**Solution**: Carefully review the route configuration for the specified path ('${your route}') and ensure it is correctly defined with the required properties. Check for missing or extra commas.
```

Comment:
Should we include a "bad" code sample here?

Reply:
Added error examples for all of the incompatible configurations conflict scenarios